### PR TITLE
feat: provide organization interface

### PR DESCRIPTION
**Pull request by skoch-intershop**

### CODE REVIEW

File: projects/organization-management/src/app/models/organization/organization.mapper.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    if (!data) {
+      throw new Error(`'organizationDocument' is required`);
+    }
+
+    return {
+      id: data.id,
+      name: data.attributes ? data.attributes.name : undefined,
+      description: data.attributes ? data.attributes.description : undefined,
+      authenticationUrl: data.attributes ? data.attributes.authenticationUrl : undefined,
+      nodes: data.relationships.nodes.data.map(identifier => identifier.id),
```
Comment (jometzner) [inline projects/organization-management/src/app/models/organization/organization.mapper.ts:17]: Shouldn't that refer to the node.mapper.ts? So you could work with a node-tree for the given organization?

Comment (skoch-intershop) [reply projects/organization-management/src/app/models/organization/organization.mapper.ts:17]: Request contains not enough information to build up an node tree.

File: projects/organization-management/src/app/models/organization/organization.mapper.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  static fromData(payload: OrganizationDocument): Organization {
+    const data = payload.data;
+
+    if (!data) {
+      throw new Error(`'organizationDocument' is required`);
+    }
+
+    return {
+      id: data.id,
+      name: data.attributes ? data.attributes.name : undefined,
```
Comment (jometzner) [inline projects/organization-management/src/app/models/organization/organization.mapper.ts:14]: Please use safe navigation with `?.` operator

Comment (jometzner) [reply projects/organization-management/src/app/models/organization/organization.mapper.ts:14]: ALso `id` and `name` are both mandatory attributes according to the API description

**Pull request merged**